raiden/transfer/mediated_transfer/state_change.py

```diff
@@ -870,3 +866,39 @@ def from_dict(cls, data: Dict[str, Any]) -> "ActionSendUnlockLight":
             receiver=to_canonical_address(data["receiver"])
         )
         return instance
+
+
+class StoreRefundTransferLight(StateChange):
+    """ Initial state of a refund transfer reception.
+
+    Args:
+        transfer: a message object that represents the refund transfer sent to a light client
+    """
+
+    def __init__(
+        self, transfer: RefundTransfer
+    ) -> None:
+        if not isinstance(transfer, RefundTransfer):
+            raise ValueError("transfer must be an RefundTransfer instance.")
+
+        self.transfer = transfer
+
+    def __repr__(self) -> str:
+        return "<StoreRefundTransferLight transfer:{}>".format(self.transfer)
+
+    def __eq__(self, other: Any) -> bool:
+        return (
+            isinstance(other, StoreRefundTransferLight)
+            and self.transfer == other.transfer
+        )
+
+    def __ne__(self, other: Any) -> bool:
+        return not self.__eq__(other)
+
+    def to_dict(self) -> Dict[str, Any]:
+        return {"transfer": self.transfer.to_dict()}
+
+    @classmethod
+    def from_dict(cls, data: Dict[str, Any]) -> "StoreRefundTransferLight":
+        return cls(transfer=data["transfer"])
```

Comment:
I don't know if this works out of the box, i mean maybe you need to construct this using the RefundTransfer.from_dict method, something like this ```return cls(transfer=RefundTransfer.from_dict(data["transfer"]))``` for example

Reply:
nice catch!